exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Some of this information would be really helpful up in the introduction! For example, introducing that we will be using data from IUE and maybe one sentence about what it is. A link to a mission webpage for an interested user would also help. </p><p> </p><p>For the heads-up, as the reader new to UV, I kind of feel like "Oh no, how I am supposed to know the correct name of my target?" Is there a standardized list somewhere, or a name resolver, that you can link to? In other words, how did you know the correct name? (What would happen if you put the "z" in the name, would it still work?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>